src/repr/src/relation.rs

```diff
@@ -289,6 +289,11 @@ impl ColumnName {
     pub fn as_mut_str(&mut self) -> &mut String {
         &mut self.0
     }
+
+    /// Returns if this [`ColumnName`] is similar to the provided one.
+    pub fn is_similar(&self, other: &ColumnName) -> bool {
+        self.0.to_lowercase() == other.as_str().to_lowercase()
```

Comment:
Is the plan to generalize this using some other similarity measure in the future?

Reply:
it should be very easy to add https://docs.rs/strsim/0.10.0/strsim/fn.levenshtein.html as well, in my experience case-insensitive + levenshtein works well

Reply:
The idea is that we could use something like @guswynn mentioned. I omitted it for now because I figured we could start with just case sensitivity